File: src/easy/minutes_to_seconds.rs
pub fn minutes_to_seconds(minutes: i32) -> i32 {
    minutes * 60
}
#[cfg(test)]
mod tests {
    use super::minutes_to_seconds;
    #[test]
    fn test_minutes_to_seconds() {
        assert_eq!(minutes_to_seconds(6), 360);
        assert_eq!(minutes_to_seconds(4), 240);
        assert_eq!(minutes_to_seconds(8), 480);
        assert_eq!(minutes_to_seconds(60), 3600);
    }
}

File: src/easy/minutes_to_seconds.zig
const expectEqual = @import("std").testing.expectEqual;

// @copyright https://edabit.com/challenge/8q54MKnRrm89pSLmW
pub fn minutesToSeconds(minutes: i32) i32 {
    return minutes * 60;
}
test "Convert 6 minutes to seconds" {
    try expectEqual(minutesToSeconds(6), 360);
}
test "Convert 4 minutes to seconds" {
    try expectEqual(minutesToSeconds(4), 240);
}
test "Convert 8 minutes to seconds" {
    try expectEqual(minutesToSeconds(8), 480);
}
test "Convert 60 minutes to seconds" {
    try expectEqual(minutesToSeconds(60), 3600);
}
